R/schambach_table.R

```diff
@@ -0,0 +1,157 @@
+#' @title schambach_table
+#' 
+#' @description Given grade level, shows summary table for provided subgroups.
+#' 
+#' @param mapvizieR_obj mapvizieR object
+#' @param measurementscale_is target subject
+#' @param grade target grade
+#' @param subgroup_cols what subgroups to explore, default is by school
+#' @param pretty_names nicely formatted names for the column cuts used above.
+#' @param start_fws starting season
+#' @param start_academic_year starting academic year
+#' @param end_fws ending season
+#' @param end_academic_year ending academic year
+#' @param complete_obsv if TRUE, limit only to students who have BOTH a start
+#' and end score. default is FALSE.
+#' 
+#' @export
+
+schambach_table_1d <- function(
+  mapvizieR_obj, 
+  measurementscale_is,
+  grade,
+  subgroup_cols = c('end_schoolname'),
+  pretty_names = c('School Name'),
+  start_fws,
+  start_academic_year,
+  end_fws,
+  end_academic_year,
+  complete_obsv = FALSE
+) {
+  
+#   if (missing(subgroup_cols) | missing(pretty_names)) {
```

Comment:
is this made redundant by the default values for `subgroup_cols`?  if so let's kill it